odl/discr/lp_discr.py

```diff
@@ -699,7 +692,7 @@ def ufunc(self):
         These are optimized to use the underlying ntuple space and incur no
         overhead unless these do.
         """
-        return DiscreteLpUFuncs(self)
+        return DiscreteLpElementUfunc(self)
```

Comment:
Nah, perhaps overkill. But the object sits on the element, not the space, that was the reasoning.